Redirects - move Filters from left to [Search Tools] remove sidebar

pe7er opened the pull request:
I had an "HEAD detached" problem & some merge conflicts with #6953 that I was not able to resolve.
This new PR removes the left sidebar as discussed in #6953.

---

This PR moves the Filter option of the **Redirect Manager: Links** to the [Search Tools] button.
(See Issue #6941 regarding the inconsistency with Filter options).
## Test instructions

In back-end > Components > Redirects
See the filter "- Select Status-" on the left
Also notice that the Expired URL column is too small.

![screen shot 2015-05-15 at 08 46 30](http://issues.joomla.org/uploads/1/f6b195be88171b43f16976bc973b629c.png)

Compare with the other filters, e.g. in **Category Manager**: 
In back-end > Content > Categories
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.
This PR also fixes the **column width** of the **Expired URL** column (now the columns Expired URL, New URL, and Referring Page are all 30%) so that it looks better on a smaller screen.

![screen shot 2015-05-15 at 08 46 30](http://issues.joomla.org/uploads/1/29d83a28b3f68d79f5bc1a167b11c29c.png)

pe7er committed:
new PR to replace #6953 with sidebar removed

zero-24 commented on administrator/components/com_redirect/views/links/tmpl/default.php:
```diff
@@ -22,28 +22,9 @@
-			</div>
-		</div>
 		<div class="clearfix"> </div>
+		<?php echo JLayoutHelper::render('joomla.searchtools.default', array('view' => $this)); ?>
```
@pe7er maybe we should move this one line above so we have

```
<?php echo JLayoutHelper::render('joomla.searchtools.default', array('view' => $this)); ?>
<div class="clearfix"> </div>
```

Else the red info box is to near to the searchtools ;)

zero-24 commented:
Works ok here except the inline comment.

pe7er committed:
moved clearfix div

pe7er committed:
moved clearfix div

pe7er committed:
moved clearfix div

pe7er commented:
I moved the clearfix div, thanks @zero-24 !

watchfulli-dev commented:
@test success after applying the patch the filters from left side now appear on top under "Search tools". One little improvement maybe can be added by adding more space between search tools area and the alert box, check attached print screen! there should be some more space on top of the red area [screen shot 2015-08-21 at 06 19 13](http://issues.joomla.org/uploads/1/2a9df4fd469cf8023934dfc805577ccc.png)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7697">issues.joomla.org/joomla-cms/7697</a>.</sub>

pe7er committed:
added a <br/> for more space between filter + message

pe7er commented:
I've added a <br/> to fix the space between the Search Tools area and the alert box.
Previous
![redirect-before](https://cloud.githubusercontent.com/assets/1217850/9490900/845c6db6-4beb-11e5-9394-28ae2b2c9e89.png)

Fixed:
![redirect-after](https://cloud.githubusercontent.com/assets/1217850/9490899/84476b28-4beb-11e5-9156-c1b27318d62c.png)

Thanks for testing & your suggestion for improvement @watchfulli-dev !

zero-24 commented:
Thanks! Works great. RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7697">issues.joomla.org/joomla-cms/7697</a>.</sub>

roland-d commented:
Thanks everybody. Merged into 3.5-dev with commit  d9f5026
